sources/packages/web/src/views/institution/locations/programs/LocationProgramView.vue

```diff
@@ -1,37 +1,108 @@
 <template>
   <v-container>
-    <h5 class="color-grey">View Program</h5>
+    <h5 class="color-grey"><a @click="goBack()">&#60;- Back to Programs</a></h5>
```

Comment:
instead of code , you can use icons -> https://materialdesignicons.com/icon/arrow-left-drop-circle

Reply:
Done